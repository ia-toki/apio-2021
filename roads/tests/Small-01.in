c10234cabdfd6eae47773fb6f163e0350fc797e1
150
107 123 877656381
75 23 954037632
23 0 751950653
15 3 886375777
18 44 979176984
59 26 735548230
94 75 977809165
12 93 589331204
116 149 874044974
40 42 998026262
99 142 937689560
2 121 955527621
6 16 696503256
137 136 966670903
93 30 543967244
148 18 561801873
145 144 729727143
104 125 867239910
72 127 751405388
108 0 923535008
29 133 864268059
92 83 596414766
25 91 795281821
0 77 795473196
45 49 754340973
3 143 601972169
123 39 933257200
118 131 770217775
61 108 915791816
44 91 738580140
65 23 789258056
68 103 591476762
85 79 531932885
63 41 633995616
90 75 635847185
100 90 650067623
106 95 945541783
26 52 540285021
53 15 940520867
22 102 729906529
64 127 592100619
39 66 618292103
33 49 556890970
46 111 673898921
76 39 726723395
9 99 518629602
11 33 972642710
102 120 824749864
97 5 834855079
89 18 938909215
87 48 701431840
70 47 895796341
50 49 594499715
113 87 537869362
111 138 879728035
62 143 804319487
57 64 770323206
67 1 845750736
52 10 516986391
83 60 678461946
71 118 873038444
144 15 686039392
13 24 623886547
127 54 935426036
120 81 725931076
140 95 691961765
141 8 584024532
31 109 753561905
28 75 530272995
36 25 805625238
114 104 582382883
19 99 943791112
139 56 895867505
54 13 846390011
24 61 915124427
126 20 538773821
55 22 582778200
98 37 861153159
80 2 558106785
112 29 630248695
132 21 768687340
34 28 945106418
129 94 802247030
119 17 627334554
1 94 965540866
105 20 736756501
134 57 697839458
27 61 903288354
17 3 945540866
81 84 591427753
88 14 779905183
122 25 917208551
30 99 991171798
32 12 919350633
125 107 536295441
4 22 933754166
5 23 801235520
41 22 858471220
74 23 612379733
147 79 982035606
51 141 924774813
103 32 987816438
20 35 929917305
86 33 660723604
96 36 660881000
66 21 614860610
42 101 914046967
82 81 573254949
79 78 693737715
78 89 819056739
128 46 638551874
60 59 718176420
84 75 904345730
43 28 788532979
16 85 914769222
48 0 675324632
37 2 762908724
35 107 974611472
69 37 798449954
143 47 523518058
149 58 705875560
135 30 870729102
136 61 852654798
14 114 576158110
138 125 698481114
110 72 509574186
21 63 566925964
130 31 579616529
49 44 778558408
73 23 927019405
91 132 644376574
7 20 596939305
142 97 812108489
146 101 755387095
10 117 715237021
47 92 659053534
124 96 642212719
8 135 887504735
131 83 528152029
95 114 575384474
56 52 807701380
101 87 753305616
133 68 675966207
117 100 784581643
121 99 721673779
58 111 663918810
109 99 585322828
38 119 517509202
115 138 956768964
