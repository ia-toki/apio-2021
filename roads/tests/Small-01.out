064bde9ff69ddc34b3b45c2d26d58873d85290d3
OK
112502233251 61480101900 23849866017 7205365011 2670369795 1131009335 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
